symphony/lib/core/class.datetimeobj.php

```diff
@@ -17,10 +17,15 @@ class DateTimeObj
      * Holds the various settings for the formats that the `DateTimeObj` should
      * use when parsing input dates.
      *
+     * @since Symphony 2.7.0 it contains default values to prevent the case where
+     * no settings are set
      * @since Symphony 2.2.4
      * @var array
      */
-    private static $settings = array();
+    private static $settings = array(
+        'time_format' => 'g:i a',
+        'date_format' => 'm/d/Y',
+    );
```

Comment:
Yup, it is already there at https://github.com/symphonycms/symphony-2/pull/2705/files#diff-5420cf877b2e40bf5839ca454c3544bcR92